src/main/java/frc/robot/subsystems/Module.java

```diff
@@ -87,6 +105,19 @@ private void setSpeed(SwerveModuleState desiredState, boolean isOpenLoop) {
       m_driveMotor.set(ControlMode.Velocity, velocity, DemandType.ArbitraryFeedForward,
           feedforward.calculate(desiredState.speedMetersPerSecond));
     }
+    if(Constants.kLogging){
+      LogManager.addDoubleArray("Swerve/Modules/"+m_moduleAbbr+"/DriveValues",
+        new double[]{
+          desiredState.speedMetersPerSecond, 
+          Conversions.falconToMPS(m_driveMotor.getSelectedSensorVelocity(), DriveConstants.kWheelCircumference,
+            DriveConstants.kDriveGearRatio),
+          Conversions.falconToMPS(m_driveMotor.getSelectedSensorVelocity(), DriveConstants.kWheelCircumference,
+            DriveConstants.kDriveGearRatio)-desiredState.speedMetersPerSecond,
+          m_driveMotor.getMotorOutputVoltage(), 
+          m_driveMotor.getStatorCurrent()
+        }
```

Comment:
don't log all this stuff in one array. Also, this is probably pinging the motors too much and will cause issues